libs/openFrameworks/math/ofMath.h

```diff
@@ -284,7 +284,7 @@ float ofRadToDeg(float radians);
 ///
 /// \param degrees An angle in degrees.
 /// \returns the angle in radians.
-float ofDegToRad(float degrees);
+float ofDegToRad(const float & degrees);
```

Comment:
will this &ref not remove the possibility of writing directly 

```c++
auto a = ofDegToRad(90);
```

Reply:
totally agreed. reverted and pushed a commit